specification/programmableconnectivity/ProgrammableConnectivity.Management/Gateway.tsp

```diff
@@ -51,6 +51,312 @@ model GatewayProperties {
+model Person {
+  /**
+   * Family name of the contactable person.
+   */
+  familyName: string;
+
+  /**
+   * Given name of the contactable person.
+   */
+  givenName: string;
+
+  /**
+   * Email address of the contactable person.
+   */
+  emailAddress: string;
+}
+
+/**
+ * Model defining a geographic address.
+ */
+@TypeSpec.Versioning.added(Versions.v2024_11_01_preview)
+model GeographicAddress {
+  /**
+   * Street Number of the address.
+   */
+  streetNumber?: string;
+
+  /**
+   * Street Name of the address.
+   */
+  streetName?: string;
+
+  /**
+   * Locality of the address.
+   */
+  locality?: string;
+
+  /**
+   * City of the address.
+   */
+  city?: string;
+
+  /**
+   * Postal code of the address.
+   */
+  postalCode?: string;
+
+  /**
+   * Country code of the address.
+   */
+  @pattern("[A-Z]{2}", "Provide 2 letter country code")
+  countryCode: string;
+}
+
+/**
+ * Model defining a local representative for a country/region.
+ */
+@TypeSpec.Versioning.added(Versions.v2024_11_01_preview)
+model LocalRepresentative {
+  /**
+   * Country code for the country/region the local representative is representing.
+   */
+  @pattern("[A-Z]{2}", "Provide 2 letter country code")
+  countryCode: string;
+
+  /**
+   * Local representative for this country/region.
+   */
+  localRepresentativePerson: Person;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required" ""
+@TypeSpec.Versioning.added(Versions.v2024_11_01_preview)
+union Category {
+  agriculture: "Agriculture",
+  artAndDesign: "Art and design",
+  autoAndVehicle: "Auto and vehicle",
+  beauty: "Beauty",
+  booksAndReference: "Books and reference",
+  business: "Business",
+  construction: "Construction",
+  defense: "Defense",
+  developerTools: "Developer tools",
+  education: "Education",
+  engineering: "Engineering",
+  eventsAndEntertainment: "Events and entertainment",
+  finance: "Finance",
+  foodAndDrink: "Food and drink",
+  games: "Games",
+  healthAndFitness: "Health and fitness",
+  healthcare: "Healthcare",
+  information: "Information",
+  kids: "Kids",
+  librariesAndDemo: "Libraries and demo",
+  lifestyle: "Lifestyle",
+  manufacturing: "Manufacturing",
+  mapsAndNavigation: "Maps and navigation",
+  media: "Media",
+  medical: "Medical",
+  mining: "Mining",
+  musicAndAudio: "Music and audio",
+  newsAndMagazines: "News and magazines",
+  organizations: "Organizations",
+  other: "Other",
+  photoAndVideo: "Photo and video",
+  productivity: "Productivity",
+  publicService: "Public service",
+  realEstate: "Real estate",
+  shopping: "Shopping",
+  socialNetworkingAndCommunications: "Social networking and communications",
+  sports: "Sports",
+  tourism: "Tourism",
+  utilities: "Utilities",
+  trading: "Trading",
+  transportation: "Transportation",
+  travelAndLocal: "Travel and local",
+  water: "Water",
+  weather: "Weather",
+  string,
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required" ""
```

Comment:
Why do you need this suppression?

Reply:
As above, these are in our API because we forward them to another, industry-standard, API. There is no extra description in the industry-standard API definition so we don't want to add information; therefore, any documentation here would just add a description that is the same as the value which isn't useful. Therefore, I decided to suppress documentation required.

Reply:
Resolving this, but see my other comment.